### PR TITLE
Create Dashboard notifications page

### LINKED ISSUE #51: Add Streamlit functionality to allow users to subscribe
## Description

Add functionality into our project to allow users to subscribe to weekly PDF reports, and daily new release emails

## Required Files

N/A

## User Story

As a user, I need to be able to subscribe to these emails in order to keep updated with the regular releases of games

### PR DESCRIPTION
**Pull request by annalisev**
Creates a dashboard page where you can enter your email to subscribe to notifications about new topics.

Closes #51 .

### CODE REVIEW

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+'''A script that creates a notification setup page for our dashboard.'''
+
+from os import environ as ENV
+
+from dotenv import load_dotenv
+import streamlit as st
+from boto3 import client
+
+TAG_ARNS = {'Action': 'arn:aws:sns:eu-west-2:129033205317:c10-games-action-tag',
```
Comment (hhm970) [inline dashboard/dashboard.py:9]: Good use of global variables here

File: dashboard/dashboard.py
Comment (hhm970) [inline dashboard/dashboard.py:1]: The dashboard file looks coherent; ready to be Dockerised!

**Review by hhm970 — APPROVED** [review]

File: dashboard/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            'Adventure': 'arn:aws:sns:eu-west-2:129033205317:c10-games-adventure-tag',
+            'Indie': 'arn:aws:sns:eu-west-2:129033205317:c10-games-indie-tag',
+            'Horror': 'arn:aws:sns:eu-west-2:129033205317:c10-games-horror-tag'}
+
+
+def verify_email(email_address: str, config):
```
Comment (annalisev) [inline dashboard/dashboard.py:15]:
```suggestion
def verify_email(email_address: str, config) -> None:
```

File: dashboard/pages/newsletter.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            verify_email(email, ENV)
+            st.write("You have subscribed!")
+            st.write(f"You are now tracking: {', '.join(selected)}")
+            for topic in selected:
+                subscribe_to_topic(email, ENV, topic)
+
+    with st.sidebar:
+        st.title("Navigation Station :rocket:")
+        st.write("---")
+        st.page_link("http://localhost:8501/",
```
Comment (Eibironke) [inline dashboard/pages/newsletter.py:75]: I think this what dan said you should chnage

Comment (annalisev) [inline dashboard/pages/newsletter.py:75]:
```suggestion
```

File: dashboard/pages/newsletter.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            verify_email(email, ENV)
+            st.write("You have subscribed!")
+            st.write(f"You are now tracking: {', '.join(selected)}")
+            for topic in selected:
+                subscribe_to_topic(email, ENV, topic)
+
+    with st.sidebar:
+        st.title("Navigation Station :rocket:")
+        st.write("---")
+                     label="Subscribe to Notifications", icon="🔔")
```
Comment (annalisev) [inline dashboard/pages/newsletter.py:75]:
```suggestion
```

File: dashboard/pages/newsletter.py (hunk 3 above)
Comment (annalisev) [reply dashboard/pages/newsletter.py]: Good point! Have removed this now.

**Review by Eibironke — APPROVED** [review]
looks good 👍

**Pull request merged**